docs/content/en/docs/tasks/cluster/cluster-license.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "License cluster"
+linkTitle: "License cluster"
+weight: 20
+date: 2021-12-10
+description: >
+  How to license your cluster.
+---
+
+If you are are licensing an existing cluster, apply the following secret to your cluster:
```

Comment:
Is "my-license-here" replaced by a license number or similar? If so, maybe add "..., apply the following secret to your cluster (replacing `my-license-here` with your license number):"

Reply:
Done.